vault-core/src/test/java/org/apache/jackrabbit/vault/packaging/impl/PackagePropertiesImplTest.java

```diff
@@ -93,9 +94,6 @@ public void testDateFormat() {
         checkDateParsing("2021-05-26T15:12:21.673+0200","2021-05-26T13:12:21.673Z");
         checkDateParsing("2021-05-26T15:12:21.673-0230","2021-05-26T17:42:21.673Z");
 
-        // missing timezone is treated as UTC. TODO is that right?
-        checkDateParsing("2021-05-26T15:12:21.673","2021-05-26T15:12:21.673Z");
-
```

Comment:
What happens now if the timezone is missing? I don't know whether anything produces that date format, but at least I'm a fan of specifying such edge cases in the test, even if the documentation doesn't really say anything about that. Not sure whether it's a problem if that behavior changes. That throws an exception now, right?

Reply:
That date format was never produced and therefore is not supported (i.e. returns null)

Reply:
It didn't return null, as the test says. But never mind.